dbeam-core/src/main/java/com/spotify/dbeam/beam/BeamHelper.java

```diff
@@ -39,8 +40,24 @@
 public class BeamHelper {
   private static Logger LOGGER = LoggerFactory.getLogger(BeamHelper.class);
 
-  public static PipelineResult waitUntilDone(PipelineResult result) {
-    PipelineResult.State state  = result.waitUntilFinish();
+  public static PipelineResult waitUntilDone(PipelineResult result,
+                                             Duration exportTimeout) {
+    PipelineResult.State state = result.waitUntilFinish(
+        org.joda.time.Duration.millis(
```

Comment:
Do we need to do this `java.time.Duration` round trip to millis? Can we use only Joda?

Reply:
Joda's `Duration.parse()` does not work well for things like `"PT1M"`. Given we are targeting Java 8, I feel it is better to just use `java.time`, but Beam SDK API still uses Joda.